docker-compose.yml

```
@@ -157,7 +157,7 @@ services:
       context: .
       dockerfile: .docker/database.dockerfile
       args:
-        IMAGE: "${VORTEX_DB_IMAGE:-uselagoon/mysql-8.4:25.11.0}" # Use custom database image (if defined) or fallback to standard database image.
+        IMAGE: "${VORTEX_DB_IMAGE:-uselagoon/mysql-8.4:25.12.1}" # Use custom database image (if defined) or fallback to standard database image.
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)